packages/types/src/options.ts

```diff
@@ -152,6 +152,21 @@ export interface ClientOptions<TO extends BaseTransportOptions = BaseTransportOp
    */
   tunnel?: string;
 
+  /**
+   * Currently controls if the user id (e.g. set by `Sentry.setUser`) should
+   * be used for dynamic sampling. Only if this flag is set to `true`, the user id
+   * will be propagated to outgoing requests in the `baggage` Http header.
+   *
+   * Note that in the next major version of this SDK, this option will not only
+   * control dynamic sampling data: As long as this flag is not set to `true`,
+   * the SDK will not send sensitive data by default.
+   *
+   * Defaults to `false`.
+   *
+   * @experimental (will be fully introduced in the next major version)
```

Comment:
(Optional)

```suggestion
   * @experimental (will be fully introduced in the next major version)
   * @ignore
```

Reply:
I don't think the option should be ignored because it's something that users will have to set when they want to sample based on user_id.

Reply:
Agree